### ISSUE TITLE
VidSrcTo changed it's method

### ISSUE THREAD

**Issue by joshholly** [Ciarands/vidsrc-to-resolver#22]
Used to, you'd go to a URL like this

https://vidplay.online/mediainfo/XXJPsaoV_IHGrJopXsprSzLHJ_obkQ1C,183,190,139,188,165,146,180,143,202,139,129,155,209,126,223,196?sub.info=https%3A%2F%2Fvidsrc.to%2Fajax%2Fembed%2Fepisode%2FoPBkO8hl%2Fsubtitles&t=4xjSCPUkAFYAyw%3D%3D&ads=0&src=vidsrc&autostart=true

And it will return the sources. Now the mediainfo endpoint looks like this:

https://vid2v11.site/mediainfo/CbpeT5vdYhPZtbmd?t=4xjSCPQgBVUOzw%3D%3D&ads=0&src=vidsrc&autostart=true&h=zmK-SfuRcp3noB6_

And returns a response like this

```
{
  "status": 200,
  "result": "Y_uHUcYdzfpZhOLdgSEykoSfgj48Xv2j2YnL9PNMcUnZ7_TgvEZVFP2tooxze2BuEpC_EOJGvQbqfyVJ8Ftat-o-EtnDbNK-4RNhJcmg5LuffA1dMgWxP4CIM7_Cqv_u4Zz-TasDtPTnrpcaJP3oCiYXcf88bhGTyTgGCbqw8FA9wv43EfM426m_TmrqX5CZWqINm5w5Wv73ZTedVLchKxnGapRXpnJmu5qhQabu78Njow4f76fN68eId7m0LEGGieUzsTAnWNaW1ONaYzr30RVi3bs6xVkDLgMOYGt7PXZJOu2w-z84Qv8rvr0vvucE623IkXlfAzf4cBC6eIqPEdV2T6y3gOXVFQkm2-dZjsyEKp7BdgQT7ourq4wsRUVqCt6vE_o98wbBaPDRP1LCSL4Lh0MZxf57Amzh9pdu0Y8h1XBr-tTC_DIH0i1SIIgR5JXzerBH2hDhd1kzPsdm8Z9beTtGs7tjqYFuhFk7HC2CEEIJ9alYwEjSndizhMK3zRacYhUhi8DYgNOvi2geOUZ396H6DsVimzSDdc9_xVBogBwLAkvWP2NelTI6PJd9-0EqbGw-JLpQal0WKarzbXmWgtHi1NYLvMlwL1COnVz7M7N6YZccZpm_ZnAlFdiJjARYkRR1uHzN0hp5sl6EhIqcIWS-lVsUIHMlyQLqT8LN0t5e-HRlM6jm8H_q_kkubQ2JQRMDgE_6KRYPdhiA3sMwWgUYnNjodIH64ratWwgGSnbWEh9gYhLqIRVqHNLLH7vPY2Tm1ZOwmaLN13OoSe972VKBf_OtGSYOPuqHNPl2wE59h60MF84hLQ_Asv12Y_2NaQfTrJHwGmbroHDOBrBq8FCNIkDjkzyv5dwtz_FCRuGK3mLns7oRxUyR1k1FIIzweIasmFLm9QNk73WEK9Uy_2gC8oK6J9xre7ejYqEZnrxc8v4VQ2_wUonrNmahnDv_ZADFypmxiDy9gIogisiJfTeojs7BCsQzEYsjBgzEVorPzWiTWTr7shuNb8fb-q03XvF_mjz2qDoh4bpO1VetQzRQyjRe3Ol80A=="
}
```

**Comment by heidan123** [comment]
yes, please fix .thanks

**Comment by Ciarands** [comment]
Resolved by #23
